packages/image/README.md

```diff
@@ -0,0 +1,132 @@
+# `@sveltejs/image`
+
+**WARNING**: This package is experimental. It uses pre-1.0 versioning and may introduce breaking changes with every minor version release.
+
+This package aims to bring a plug and play image component to SvelteKit that is opinionated enough so you don't have to worry about the details, yet flexible enough for more advanced use cases or tweaks. It serves a smaller file format like `avif` or `webp` and uses the `srcset` and `sizes` attributes of the `img` tag to provide resized images suitable for various device sizes, which for example results in smaller images downloaded for mobile.
+
+## Setup
+
+Install:
+
+```bash
+npm install --save @sveltejs/image
+```
+
+Adjust `vite.config.js`:
+
+```diff
++import { images } from '@sveltejs/image/vite';
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	plugins: [
++		images({ providers: { default: '@sveltejs/image/providers/<choose one>' } }),
```

Comment:
this is wrong, it's missing the `runtime` portion:

```suggestion
+		images({
			runtime: {
				providers: {
					default: '@sveltejs/image/providers/<choose one>'
				}
			}
		}),
```

when you add it, the config starts to get _super_ unwieldy for the simple case. i do have to wonder if we really need multi-provider support, it's the edgiest of edge cases. If we _really, really_ needed it then maybe something like this would be neater?

```svelte
<script>
  const my_provider = {
    getSrc: ...
  };
</script>

<Image ... provider={my_provider} />
```

But to be clear I don't think that's something we need for an initial release. Let's just simplify the hell out of it

Reply:
How would the simple provider version look like then?

We also have to think about provider options which you might want to pass, like the default quality used for Vercel image optimization.